.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode9.4
+osx_image: xcode10.2
 before_install:
   - rvm get head
```

Comment:
Could you please remove this entire `before_install` block? We have had issues in the past with `rvm get head` breaking travis builds. Thanks!